WooCommerce/Classes/POS/ViewModels/PointOfSaleDashboardViewModel.swift

```diff
@@ -15,6 +15,8 @@ final class PointOfSaleDashboardViewModel: ObservableObject {
 
     @Published var showsCardReaderSheet: Bool = false
     @Published private(set) var cardPresentPaymentEvent: CardPresentPaymentEvent = .idle
+    @ObservedObject private(set) var cardReaderConnectionViewModel: CardReaderConnectionViewModel
```

Comment:
Who owns this? Since we're creating it in the init, perhaps this VM does? Having it as an observed object here and in the `CardReaderConnectionStatusView` may lead to some strange situations, depending on the ownership model of this VM.

Perhaps we should have the view own it, as a StateObject?

Reply:
Great question, I'm thinking about keeping it as a let as it makes sense to be the same life cycle as `PointOfSaleDashboardViewModel`. I'll update it in a separate PR.

Reply:
Should it also be an @StateObject here then, and remain @ObservedObject in the connection view? I'm still not totally clear but I think that's the approach.